lib/matplotlib/tests/test_axes.py

```diff
@@ -5782,3 +5782,46 @@ def test_zoom_inset():
                    [0.8425, 0.907692]])
     np.testing.assert_allclose(axin1.get_position().get_points(),
             xx, rtol=1e-4)
+
+
+def test_inset_codes():
+    "Test that the inset codes put the inset where we want..."
```

Comment:
Single double-quotes as function docs are pretty non-standard. Not sure if it matters for tests though.